AV-101448 - Documentation update all datasources (#328)
couchbasecloud/terraform-provider-couchbase-capella: docs/data-sources/backups.md

page_title: "couchbase-capella_backups Data Source - terraform-provider-couchbase-capella"
subcategory: ""
description: |-
  Manages backup datasource, retrieves backups associated with a bucket for a Couchbase Capella cluster.
---

# couchbase-capella_backups (Data Source)

> Manages backup datasource, retrieves backups associated with a bucket for a Couchbase Capella cluster.

Reviewer: ```suggestion
The backups data source retrieves backups associated with a bucket for an operational cluster.
```
